libs/utils/utils.js

```diff
@@ -428,6 +428,20 @@ export function b64ToUtf8(str) {
   return decodeURIComponent(escape(window.atob(str)));
 }
 
+const RE_ALPHANUM = /[^0-9a-z]/gi;
+const RE_TRIM_UNDERSCORE = /^_+|_+$/g;
+export const analyticsGetLabel = (txt) => txt.replaceAll('&', 'and').replace(RE_ALPHANUM, '_').replace(RE_TRIM_UNDERSCORE, '');
+
+export const analyticsDecorateList = (li, idx) => {
+  const link = li.firstChild?.nodeName === 'A' && li.firstChild;
+  if (!link) return;
+
+  const label = link.textContent || link.getAttribute('aria-label');
+  if (!label) return;
+
+  link.setAttribute('daa-ll', `${analyticsGetLabel(label)}-${idx + 1}`);
+};
+
```

Comment:
What do you think about putting this in analytics.js? We have a decorateBlock, decorateLinks, etc. and this would slot in nicely.

Reply:
Looks like analytics.js isn't merged yet (part of consonant branch).  I'll move the fn's over when that gets merged.